src/uu/ls/src/ls.rs

```diff
@@ -1344,14 +1345,25 @@ fn pad_left(string: String, count: usize) -> String {
 
 fn display_items(items: &[PathData], config: &Config, out: &mut BufWriter<Stdout>) {
     if config.format == Format::Long {
-        let (mut max_links, mut max_size) = (1, 1);
+        let (mut max_links, mut max_width) = (1, 1);
+        let mut total_size = 0;
+
         for item in items {
-            let (links, size) = display_dir_entry_size(item, config);
+            let (links, width) = display_dir_entry_size(item, config);
             max_links = links.max(max_links);
-            max_size = size.max(max_size);
+            max_width = width.max(max_width);
+            total_size += item.md().map_or(0, |md| display_block_size(md, config));
         }
+
+        #[cfg(unix)]
+        {
+            if total_size > 0 {
+                let _ = writeln!(out, "total {}", display_file_size(total_size, config));
```

Comment:
On my machine, I just get the number of blocks without any additional formatting so `10000` instead of `10K`. This returns the latter right?